debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PGTableSchemaBuilder.java

```diff
@@ -293,6 +293,7 @@ protected StructGenerator createValueGenerator(Schema schema, TableId tableId, L
                   result.put(fields[i], null);
                 }
               } else {
+                value = converter.convert(value);
```

Comment:
in case of YBOutout can we throw error! This class shouldn't be used in case if yboutput